astropy/extern/ply/lex.py

```diff
@@ -41,8 +41,6 @@
 import os
 import inspect
 
-from ...extern.six.moves import zip
```

Comment:
L45 below, they look for the python version for handling the strings, and for the zip it's not necessary either.